packages/cli/src/commands/eject/index.ts

```diff
@@ -0,0 +1,175 @@
+import chalk from 'chalk';
+import fs from 'node:fs';
+import path from 'node:path';
+import type { Arguments } from 'yargs-parser';
+import { pkg } from '../../pkg.js';
+import { DEFAULT_VALUES, type EjectOptions } from './options.js';
+import { errorLabel, printHelp, primaryLabel } from '../../utils/messages.js';
+import { parseAstroConfig, replaceArgs, generateAstroConfig } from '../../utils/astro-config.js';
+import { updateWorkspaceVersions } from '../../utils/workspace-version.js';
+
+interface PackageJson {
+  dependencies: Record<string, string>;
+  devDependencies: Record<string, string>;
+}
+
+const TUTORIALKIT_VERSION = pkg.version;
+const REQUIRED_DEPENDENCIES = ['@tutorialkit/runtime', '@webcontainer/api', 'nanostores', '@nanostores/react'];
+
+export function ejectRoutes(flags: Arguments) {
+  if (flags._[1] === 'help' || flags.help || flags.h) {
+    printHelp({
+      commandName: `${pkg.name} eject`,
+      usage: '[folder] [...options]',
+      tables: {
+        Options: [
+          [
+            '--force',
+            `Overwrite existing files in the target directory without prompting (default ${chalk.yellow(DEFAULT_VALUES.force)})`,
+          ],
+        ],
+      },
+    });
+
+    return 0;
+  }
+
+  try {
+    return _eject(flags);
+  } catch (error) {
+    console.error(`${errorLabel()} Command failed`);
+
+    if (error.stack) {
+      console.error(`\n${error.stack}`);
+    }
+
+    process.exit(1);
+  }
+}
+
+async function _eject(flags: EjectOptions) {
+  let folderPath = flags._[1] !== undefined ? String(flags._[1]) : undefined;
+
+  if (folderPath === undefined) {
+    folderPath = process.cwd();
+  } else {
+    folderPath = path.resolve(process.cwd(), folderPath);
+  }
+
+  /**
+   * First we make sure that the destination has the correct files
+   * and that there won't be any files overwritten in the process.
+   *
+   * If there are any and `force` was not specified we abort.
+   */
+  const { astroConfigPath, srcPath, pkgJsonPath, astroIntegrationPath, srcDestPath } = validateDestination(
+    folderPath,
+    flags.force,
+  );
+
+  /**
+   * We proceed with the astro configuration.
+   *
+   * There we must disable the default routes so that the
+   * new routes that we're copying will be automatically picked up.
+   */
+  const astroConfig = await parseAstroConfig(astroConfigPath);
+
+  replaceArgs({ defaultRoutes: false }, astroConfig);
+
+  fs.writeFileSync(astroConfigPath, generateAstroConfig(astroConfig));
+
+  /**
+   * We copy all assets from the `default` folder into the `src` folder.
+   */
+  fs.cpSync(srcPath, srcDestPath, { recursive: true });
+
+  /**
+   * Last, we ensure that the package.json contains the extra dependencies.
+   * If any are missing we suggest to install the new dependencies.
+   */
+  const pkgJson: PackageJson = JSON.parse(fs.readFileSync(pkgJsonPath, 'utf-8'));
+  const astroIntegrationPkgJson: PackageJson = JSON.parse(
```

Comment:
Nit: Newline above since the `JSON.parse` spans over multiple lines.

Reply:
```suggestion

  const astroIntegrationPkgJson: PackageJson = JSON.parse(
```